src/main/groovy/org/reficio/p2/resolver/eclipse/impl/FileBinaryCategory.groovy

```diff
@@ -18,6 +18,9 @@
  */
 package org.reficio.p2.resolver.eclipse.impl
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+@edu.umd.cs.findbugs.annotations.SuppressFBWarnings("SE_NO_SERIALVERSIONID")
```

Comment:
Now you have an import and don’t use it ;)

Reply:
hopefully fixed